src/lib.rs

```diff
@@ -173,6 +173,16 @@ impl Default for RtsCamera {
     }
 }
 
+impl RtsCamera {
+    /// Resets the smoothing of the camera.
+    pub fn reset_smoothing(&mut self) {
```

Comment:
I don't think 'reset' is a good word to use here. Let's call this `snap_to_target`. And update the doc to:

```
/// Snaps the camera's focus (pan), rotation, zoom, and angle to the target values, effectively skipping the usual smoothing.
```

Reply:
That's the exact same name Godot uses for [Camera2D](https://docs.godotengine.org/en/stable/classes/class_camera2d.html#class-camera2d-method-reset-smoothing) as Camera3D doesn't have such smoothing feature. Don't you think users will confuse `snap_to_target` with the target snapping feature (`snap = true`)?

Reply:
Fair point, keep `reset_smoothing`. But update the doc to be similar to (or the same as) Godot's, it's much more clear.

Reply:
Alright, what do you think about something more like this?

Reply:
Looks good